infra/cifuzz/cifuzz.py

```diff
@@ -196,14 +197,19 @@ def run_fuzzers(fuzz_seconds, workspace, project_name):
     logging.error('No fuzzers were found in out directory: %s.',
                   format(out_dir))
     return False, False
-  fuzz_seconds_per_target = fuzz_seconds // len(fuzzer_paths)
 
   # Run fuzzers for alotted time.
+  fuzzers_left = len(fuzzer_paths)
   for fuzzer_path in fuzzer_paths:
-    target = fuzz_target.FuzzTarget(fuzzer_path, fuzz_seconds_per_target,
-                                    out_dir, project_name)
+    run_seconds = fuzz_seconds // fuzzers_left
+    if run_seconds <= 0:
```

Comment:
why would this happen? looks like we can skip some fuzzers in such case? it's especially concerning given that lines 210-212 use the wall clock and I suspect the `elapsed_time` can be greater than expected in some cases

let's do something like `run_seconds = max(run_seconds, <DEFAULT_VALUE_FROM_THE_CONFIG>)` on this line to make sure we always run the fuzzers at least as long as stated in the config

Reply:
Good idea, just added.